docs/src/components/footer/nci-big.jsx

```diff
@@ -47,22 +47,15 @@ export const NCIBigFooter = () => {
 												} `}
```

Comment:
Why is this file being modified? This does not seem related to the navigation ticket, so please remove.